syntax = "proto3";

package coordinator;

message HeartbeatRequest {
    string workerId = 1;
}

message TaskResponse {
    string ack = 1;
    string taskId = 2;
    string workerId = 3;
}

service CoordinatorService {
    rpc HeartbeatStream (stream HeartbeatRequest) returns (stream TaskResponse);


}